### PR TITLE
[PT FE]: support aten::glu and aten::sigmoid_

### PR DESCRIPTION
**Pull request by eaidova**
### Details:
 - *support glu and inplace sigmoid activations in pytorch frontent*
 - *added tests for sigmoid*

### Tickets:
 - *101112*
 - *101126*

### CODE REVIEW

File: src/frontends/pytorch/src/op/glu.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
```
Comment (mvafin) [inline src/frontends/pytorch/src/op/glu.cpp:6]: Same comment about opset vs operation include

Comment (eaidova) [reply src/frontends/pytorch/src/op/glu.cpp:6]: done

Comment (mvafin) [reply src/frontends/pytorch/src/op/glu.cpp:6]: I still see you are using opset

File: tests/layer_tests/pytorch_tests/test_glu.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            def forward(self, x):
+                return F.glu(x, self.dim)
+
+        ref_net = None
+
+        return aten_glu(dim), ref_net, "aten::glu"
+
+    @pytest.mark.nightly
+    @pytest.mark.precommit
+    @pytest.mark.parametrize("dim", [0, 1, 2, 3, -1, -2])
```
Comment (mvafin) [inline tests/layer_tests/pytorch_tests/test_glu.py:32]: Could you also add `None` case?

File: src/frontends/pytorch/src/op/glu.cpp (hunk 1 above)
Comment (eaidova) [reply src/frontends/pytorch/src/op/glu.cpp:6]: oops, looks like I forgot to push :)

File: tests/layer_tests/pytorch_tests/test_glu.py (hunk 2 above)
Comment (eaidova) [reply tests/layer_tests/pytorch_tests/test_glu.py:32]: Torchscript failoed, 
E       glu(Tensor input, int dim=-1) -> Tensor:
E       Expected a value of type 'int' for argument 'dim' but instead found type 'NoneType'

File: src/frontends/pytorch/src/op/glu.cpp (hunk 1 above)
Comment (eaidova) [reply src/frontends/pytorch/src/op/glu.cpp]: updated

File: src/frontends/pytorch/src/op/glu.cpp [hunk 3]
```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/sigmoid.hpp"
+#include "openvino/op/split.hpp"
+#include "utils.hpp"
```
Comment (mvafin) [inline src/frontends/pytorch/src/op/glu.cpp:10]:
```suggestion
```
Don't think you need this header here

File: src/frontends/pytorch/src/op_table.cpp [hunk 4]
```diff
@@ -4,7 +4,46 @@
 
 #include "op_table.hpp"
 
-#include "openvino/opsets/opset10.hpp"
```
Comment (mvafin) [inline src/frontends/pytorch/src/op_table.cpp:7]: Well, here using opset header actually makes sense...

Comment (mvafin) [reply src/frontends/pytorch/src/op_table.cpp:7]: Please remove changes

Comment (eaidova) [reply src/frontends/pytorch/src/op_table.cpp:7]: return back

**Review by mvafin — APPROVED** [review]

**Pull request merged**